zip-0311.rst

```diff
@@ -2,7 +2,441 @@
 
   ZIP: 311
   Title: Sapling Payment Disclosure
```

Comment:
Now that we want to include transparent inputs in the signatures, and add Orchard to this format as well:
```suggestion
  Title: Zcash Payment Disclosures
```